content/get-started.html

```diff
@@ -56,13 +56,14 @@
       <div class="leading-trailing-padding">
         <h3 class="header">Where to buy NAV</h3>
         <div class="spacer-35px"></div>
-        <div class="paragraph">You can buy NAV from centralized exchanges such as <a href="https://www.binance.com" target="_blank" rel="noopener noreferrer nofollow" class="link">Binance</a> and <a href="https://www.bittrex.com" target="_blank" rel="noopener noreferrer nofollow" class="link">Bittrex</a>, or decentralized ones like <a href="https://exchange.pancakeswap.finance" target="_blank" rel="noopener noreferrer nofollow" class="link">PancakeSwap</a>. If you wish to acquire xNAV instead, it&#x27;s possible to swap NAV to xNAV at a 1 to 1 rate using one of our wallets.</div>
+        <div class="paragraph">You can buy NAV from decentralized exchanges such as <a href="https://komodian.org/products/atomicdex-desktop-ce/" target="_blank" rel="noopener noreferrer nofollow" class="link">AtomicDEX-CE</a> and <a href="https://exchange.pancakeswap.finance" target="_blank" rel="noopener noreferrer nofollow" class="link">PancakeSwap</a>, or centralized exchanges like <a href="https://www.binance.com" target="_blank" rel="noopener noreferrer nofollow" class="link">Binance</a> and <a href="https://www.bittrex.com" target="_blank" rel="noopener noreferrer nofollow" class="link">Bittrex</a>. If you wish to acquire xNAV instead, it&#x27;s possible to swap NAV to xNAV at a 1 to 1 rate using one of our wallets.</div>
         <div class="spacer-40px"></div>
         <div class="get-started-exchange-grid">
-          <div class="get-started-exchange-content"><img src="/images/binance-logo.png" loading="lazy" height="" alt="" class="get-started-exchange-image"></div>
-          <div class="get-started-exchange-content"><img src="/images/bittrex-logo.png" loading="lazy" sizes="(max-width: 479px) 56vw, (max-width: 767px) 16vw, (max-width: 991px) 15vw, (max-width: 1279px) 13vw, (max-width: 1439px) 11vw, (max-width: 1919px) 8vw, 121.328125px" alt="" class="get-started-exchange-image"></div>
+          <div class="get-started-exchange-content"><img src="/images/AtomicDEX_Horizontal-Dark.png" loading="lazy" sizes="(max-width: 479px) 56vw, (max-width: 767px) 16vw, (max-width: 991px) 15vw, (max-width: 1279px) 13vw, (max-width: 1439px) 11vw, (max-width: 1919px) 8vw, 121.328125px" alt="" class="get-started-exchange-image"></div>
```

Comment:
- should follow the naming convention of the other logos, e.g. `atomicdex-ce-logo.png`
- it'd be good if we could find it as a vector instead (`.svg`)